### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by jamesoram**
@johnduffell @jharewinton

### CODE REVIEW

**Comment by jharewinton** [comment]
:+1:

File: build.sbt [hunk 1]
```diff
@@ -11,15 +11,15 @@ scalaVersion := "2.10.3"
 lazy val root = (project in file(".")).enablePlugins(ShipAutoPlugin)
 
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "2.0",
-  "org.seleniumhq.selenium" % "selenium-java" % "2.43.1",
+  "org.scalatest" %% "scalatest_2.10" % "2.2.4",
```
Comment (johnduffell) [inline build.sbt]: do we really want to limit people to using scala 2.10 only?

**Comment by johnduffell** [comment]
apart from the scalatest build break, great! :+1:

File: build.sbt (hunk 1 above)
Comment (jamesoram) [inline build.sbt]: We are limiting the scala version in https://github.com/guardian/scala-automation/blob/master/build.sbt#L9
Maybe it's time to re-think that?

Comment (johnduffell) [inline build.sbt]: yeah maybe we should consider doing cross builds!  at the moment we only publish 2.10.  any takers?

**Comment by jamesoram** [comment]
I've removed the scala 2.10 limitation and fixed the build.

**Comment by johnduffell** [comment]
good work! thanks =) :+1:

**Pull request merged**